apps/designer/app/designer/features/style-panel/shared/css-value-input/css-value-input.tsx

```diff
@@ -110,13 +110,18 @@ const useScrub = ({
           unit,
           value: event.value,
         });
+        inputRef.current?.blur();
+        inputRef.current?.parentElement?.setAttribute("data-state", "focus");
```

Comment:
there are 2 problems here:
1. you are accessing parent element in a way that is fragile - if we change the structure - this will suddenly start adding attribute on a wrong element
3. focus has actual meaning in the sense it is focused, but in this case you are actually unfocusing, so you are kinda trying to do something like visually-focused or something. 

I just needed the same thing [here](https://github.com/webstudio-is/webstudio-designer/pull/351/files#diff-e2bb0fb4fd1b7d2d36fb8c2c21769648fc2154d3226fae44d2e559357d648639R176)

 and called it. "active" like the active CSS pseudo selector ... I am not sure if my semantics better, wdyt?
 also I did it over prop, so to set it you would have to set it over local state

Reply:
I should probably turn it into variant="active", because its just a visual thing and `variant="ghost" active` shouldn't be possible at the same time I guess

Reply:
changed it to be variant over there

Reply:
will merge totay-tomorrow that other pr so you can use the variant

Reply:
btw this has been merged, you can set that state as a prop state="active"